@navikt/ds-css/grid/grid.css

```diff
@@ -0,0 +1,11 @@
+.navds-grid {
+  display: grid;
```

Comment:
Hvilke tanker gjør vi oss mtp IE11 og bruken av grid?
grid er vel delvis støttet der (med quirks og prefixing), men f.eks `grid-gap` er ikke støttet.

Reply:
Vi diskutert problemstillingen på et møte med designsystemet, og konklusjonen var at grid var å foretrekke selv med begrenset IE11 støtte. Koden blir svært enkel å vedlikeholde, grid er spesifikt laget for disse problemene, samtidig som IE11 ikke lenger er offisielt støttet av NAV. @andnorda, @KenAJoh har dere noe å tilføye?